### PR TITLE
docs: update required spark and python version on website doc

### PR DESCRIPTION
**Pull request by JessicaXYWang**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/about.md](https://github.com/microsoft/SynapseML/blob/248e3157c2f7b5ab6db51a91026d9bcb3aa32ec5/website%2Fdocs%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d88f2490-16ca-4f73-8d09-d638b25d57f9) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.0/about.md](https://github.com/microsoft/SynapseML/blob/248e3157c2f7b5ab6db51a91026d9bcb3aa32ec5/website%2Fversioned_docs%2Fversion-0.10.0%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1a490ba-0114-42a1-8f8b-b4af7f19cc96) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.1/about.md](https://github.com/microsoft/SynapseML/blob/248e3157c2f7b5ab6db51a91026d9bcb3aa32ec5/website%2Fversioned_docs%2Fversion-0.10.1%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0075c4ce-d692-47e9-9708-41062816f8ab) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.2/about.md](https://github.com/microsoft/SynapseML/blob/248e3157c2f7b5ab6db51a91026d9bcb3aa32ec5/website%2Fversioned_docs%2Fversion-0.10.2%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce7a72a0-a07a-49bc-b619-4ad452dbe284) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.9.4/about.md](https://github.com/microsoft/SynapseML/blob/248e3157c2f7b5ab6db51a91026d9bcb3aa32ec5/website%2Fversioned_docs%2Fversion-0.9.4%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/849d621f-b2fc-48c8-a085-e64705f36464) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.9.5/about.md](https://github.com/microsoft/SynapseML/blob/248e3157c2f7b5ab6db51a91026d9bcb3aa32ec5/website%2Fversioned_docs%2Fversion-0.9.5%2Fabout.md) | 94 | 100 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7205b276-7fe2-426e-b336-b7343780f3ca) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by JessicaXYWang** [comment]
Thank you for your feedback @Mike-Soukup

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/about.md](https://github.com/microsoft/SynapseML/blob/cab8673c2bcdbdc13311534cb105e2d8858efe07/website%2Fdocs%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b3af2a7f-dce4-41f3-bb4e-b8d62126c0b5) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.0/about.md](https://github.com/microsoft/SynapseML/blob/cab8673c2bcdbdc13311534cb105e2d8858efe07/website%2Fversioned_docs%2Fversion-0.10.0%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7ba947c3-6d22-4d05-b25d-04b6b07185e3) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.1/about.md](https://github.com/microsoft/SynapseML/blob/cab8673c2bcdbdc13311534cb105e2d8858efe07/website%2Fversioned_docs%2Fversion-0.10.1%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/decf624c-90ca-46b5-9354-6daab0cd0bb1) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.2/about.md](https://github.com/microsoft/SynapseML/blob/cab8673c2bcdbdc13311534cb105e2d8858efe07/website%2Fversioned_docs%2Fversion-0.10.2%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1b720a1f-d24e-4f2c-8c63-892f2b2e76e6) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.9.4/about.md](https://github.com/microsoft/SynapseML/blob/cab8673c2bcdbdc13311534cb105e2d8858efe07/website%2Fversioned_docs%2Fversion-0.9.4%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbc1920c-b49c-4c47-9d6b-ee63a37472df) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.9.5/about.md](https://github.com/microsoft/SynapseML/blob/cab8673c2bcdbdc13311534cb105e2d8858efe07/website%2Fversioned_docs%2Fversion-0.9.5%2Fabout.md) | 94 | 100 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2def6e66-71e2-4d9b-b3ce-b3a5fc7d7f26) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by mhamilton723 — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-0.9.5/about.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ can embed **any** web service into their SparkML models. In this vein, SynapseML
 SparkML transformers for a wide variety of [Azure Cognitive Services](https://azure.microsoft.com/en-us/services/cognitive-services/). For production grade deployment, the Spark Serving project enables high throughput,
 submillisecond latency web services, backed by your Spark cluster.
 
-SynapseML requires Scala 2.12, Spark 3.0+, and Python 3.6+.
+SynapseML requires Scala 2.12, Spark 3.2+, and Python 3.8+.
```
[the remark this answers is not in the document]
Comment (JessicaXYWang) [reply website/versioned_docs/version-0.9.5/about.md:26]: @mhamilton723 Hi Mark, thanks for the review. The first versioned doc on website is for synapse [0.9.4](https://github.com/microsoft/SynapseML/tree/e6da4d5583fd443e04cc89dfbf399b88851bb507) which requires spark 3.1. I put 3.1 there for that version. And all major versions after that is 3.2, [0.9.5](https://github.com/microsoft/SynapseML/tree/79d92d3af8268ebc5faa1bc5fedfa9f5ff7cfde4) for example.

I think the last version for spark 3.0 we support is before the first SynapseML we released.  [0.9.0](https://github.com/microsoft/SynapseML/tree/a6c7fea6dc6a9bbffcdaeef3e587e5efdb1ada50) is for spark 3.1 already. Is there a mmlspark versioned doc somewhere?

[the remark this answers is not in the document]
Comment (JessicaXYWang) [reply website/versioned_docs/version-0.9.5/about.md:26]: I use spark version in build.sbt as the spark version needed, is that correct?

**Review by mhamilton723 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/about.md](https://github.com/microsoft/SynapseML/blob/f82c19c9f4fb1351a6c189850726d59d7961f543/website%2Fdocs%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37451ee4-9781-42dc-af9b-7504f30c4ad9) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.0/about.md](https://github.com/microsoft/SynapseML/blob/f82c19c9f4fb1351a6c189850726d59d7961f543/website%2Fversioned_docs%2Fversion-0.10.0%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/affd218e-c337-488f-86b4-407c0397bacf) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.1/about.md](https://github.com/microsoft/SynapseML/blob/f82c19c9f4fb1351a6c189850726d59d7961f543/website%2Fversioned_docs%2Fversion-0.10.1%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/453b6a43-bbe6-4ad2-ae2a-418a2e02d4f4) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.10.2/about.md](https://github.com/microsoft/SynapseML/blob/f82c19c9f4fb1351a6c189850726d59d7961f543/website%2Fversioned_docs%2Fversion-0.10.2%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ae7fe09-2fb0-4368-b885-3ed437f157fd) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.9.4/about.md](https://github.com/microsoft/SynapseML/blob/f82c19c9f4fb1351a6c189850726d59d7961f543/website%2Fversioned_docs%2Fversion-0.9.4%2Fabout.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01d19023-f250-4d6c-bf3f-f200ba7bc48d) | [:white_check_mark:](# "Successfully processed") |
| [website/versioned_docs/version-0.9.5/about.md](https://github.com/microsoft/SynapseML/blob/f82c19c9f4fb1351a6c189850726d59d7961f543/website%2Fversioned_docs%2Fversion-0.9.5%2Fabout.md) | 94 | 100 | 72 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2b84b04-79fd-4d12-a422-7169946532de) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**